### ISSUE TITLE
Add more quotes 

### ISSUE THREAD

**Issue by avats-dev** [avats-dev/motivate#1]
### Add more quotes
---

For contributing guidelines, go [here](https://github.com/avats-dev/motivate#contribution).

PRs are welcome :tada:

**Comment by stanimaldev** [comment]
@avats-dev I'd like to add more quotes if you want to assign me! thanks!

**Comment by avats-dev** [comment]
Hi @stanimaldev , welcome to motivate community :tada:. We're looking forward to your contribution. 
#### This issue is open to all, and could accept multiple PRs, so it is open to everyone. Noone would be assigned for this issue. Just open a PR with quotes you want to submit and then link the PR back to this issue. Thanks.

> Feel free to ping back if you need any help. :+1:

**Comment by dijodaiju7** [comment]
Hey @avats-dev I would like to contribute to this project as a part of Hacktoberfest. Could you please assign it to me

**Comment by RushiVachhani** [comment]
I this issue still open for all ?

**Comment by shagun25** [comment]
can i contribute?

**Comment by avats-dev** [comment]
Yes @shagun25 @RushiVachhani @dijodaiju7 you all can contribute to this, thanks:tada:. This will not be assigned to anyone as it is open for everyone. Also, for hacktoberfest check out #16 and refer to it in your PRs. .:+1:
